packages/fcl/src/current-user/index.js

```diff
@@ -99,11 +100,20 @@ function notExpired(user) {
 
 async function getAccountProofData() {
   let accountProofDataResolver = await config.get("fcl.accountProof.resolver")
-
-  if (!isFn(accountProofDataResolver)) return
+  if (accountProofDataResolver == null) return
+
+  if (!isAsyncFunction(accountProofDataResolver)) {
```

Comment:
Why does it need to be a async function? Could we alternatively just check if it is a function and pass it into `Promise.resolve`?

Reply:
The intention here was to warn and push toward the recommendation from the docs which states the nonce should be generated and tracked via a backend. Think this is overkill @justinbarry?

Reply:
removed in favor of simple function check